### PR TITLE
WebDriver: Add stub for `/session/{id}/print` endpoint

### PR DESCRIPTION
**Pull request by Baitinq**
"Implements" another endpoint in https://github.com/SerenityOS/serenity/issues/15551. (half a tick less??)  @trflynn89  :^))

### CODE REVIEW

File: Userland/Libraries/LibWeb/WebDriver/Error.cpp [hunk 1]
```diff
@@ -33,6 +33,7 @@ static Vector<ErrorCodeData> const s_error_code_data = {
     { ErrorCode::NoSuchFrame, 404, "no such frame" },
     { ErrorCode::NoSuchWindow, 404, "no such window" },
     { ErrorCode::NoSuchShadowRoot, 404, "no such shadow root" },
+    { ErrorCode::NotImplemented, 501, "not implemented" },
```
Comment (linusg) [inline Userland/Libraries/LibWeb/WebDriver/Error.cpp:36]: this isn't needed, `UnsupportedOperation` exists

File: Userland/Services/WebContent/WebDriverConnection.cpp [hunk 2]
```diff
@@ -1533,6 +1533,12 @@ Messages::WebDriverClient::TakeElementScreenshotResponse WebDriverConnection::ta
     return encoded_string;
 }
 
+// 18.1 Print Page, Print Page, https://w3c.github.io/webdriver/#dfn-print-page
```
Comment (linusg) [inline Userland/Services/WebContent/WebDriverConnection.cpp:1536]: double title?

File: Userland/Services/WebDriver/Client.cpp [hunk 3]
```diff
@@ -646,4 +646,15 @@ Web::WebDriver::Response Client::take_element_screenshot(Web::WebDriver::Paramet
     return session->web_content_connection().take_element_screenshot(parameters[1]);
 }
 
+// 18.1 Print Page, https://w3c.github.io/webdriver/#dfn-print-page
+// POST /session/{session id}/print
+Web::WebDriver::Response Client::print_page(Web::WebDriver::Parameters parameters, JsonValue)
+{
+    // FIXME: Actually implement this :^)
```
Comment (linusg) [inline Userland/Services/WebDriver/Client.cpp:653]: This method is complete, the FIXME belongs in the other one

File: Userland/Services/WebDriver/Client.cpp [hunk 4]
```diff
@@ -646,4 +646,15 @@ Web::WebDriver::Response Client::take_element_screenshot(Web::WebDriver::Paramet
...
+    dbgln_if(WEBDRIVER_DEBUG, "(STUB) Handling POST /session/<session id>/print");
```
Comment (linusg) [inline Userland/Services/WebDriver/Client.cpp:654]: same

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibWeb/WebDriver/Error.cpp (hunk 1 above)
Comment (Baitinq) [reply Userland/Libraries/LibWeb/WebDriver/Error.cpp:36]: Okay, I thought there was a difference between "Unsupported" and "Not Implemented", maybe meaning that unsupported just means the engine doesn't support it by design/* and not implemented is more of a temp thing? What do you think? I have no problem in removing this if you think its unnecessary :^)

File: Userland/Services/WebDriver/Client.cpp (hunk 4 above)
Comment (linusg) [reply Userland/Services/WebDriver/Client.cpp:653]: not resolved, remove `(STUB)`

File: Userland/Libraries/LibWeb/WebDriver/Error.cpp (hunk 1 above)
Comment (linusg) [reply Userland/Libraries/LibWeb/WebDriver/Error.cpp:36]: > Indicates that a [command](https://w3c.github.io/webdriver/#dfn-commands) that should have executed properly cannot be supported for some reason.

fits perfectly, there's no reason to invent our own error code for this

**Comment by Baitinq** [comment]
@linusg ty for the review :)

**Pull request merged**